strings.js

```diff
@@ -0,0 +1,54 @@
+function reversal(inputText) {
+	var stringArray = inputText.split('');
+	var reverseString = stringArray.reverse().join('');
+	output1.innerHTML = "reversal: " + reverseString;
+}
+
+function alphabits(inputText) {
+	var stringArray = inputText.split('');
+	var alphaString = stringArray.sort().join('');
+	output2.innerHTML = "alphabits: " + alphaString;
+}
+
+function palindrome(inputText) {
+	var stringArray = inputText.split('');
+	var reverseArray = stringArray.reverse().join('');
+	if (inputText===reverseArray) {
+		output3.innerHTML = "Your string is a palindrome";
+	} 
+	else {
+		output3.innerHTML = "";
+	}
+
+}
+
+
+var testString = "";
+var textBox = document.getElementById("uInput");
+var submitButton = document.getElementById("submitInfo");
+var output1 = document.getElementById("reversalOutput");
+var output2 = document.getElementById("alphaOutput");
+var output3 = document.getElementById("palindromeOut");
+var formElement=document.getElementById("submission");
+formElement.addEventListener("submit", function(e){
+	e.preventDefault();
+})
+
+function whichKey() {
+	if(event.keyCode===13) {
+		document.getElementById("submitInfo").click();
+	}
+}
+submitButton.addEventListener("click", textEntered)
+textBox.addEventListener("keyup", whichKey);
+function textEntered () {
+	var testString = textBox.value;
```

Comment:
I would rename this from testString to something more certain about what it is doing finalOutput (or something that takes the "test" away so the people who review this don't wonder if it is working